Reader/Reader.php

```diff
@@ -23,6 +23,9 @@
 
 class Reader
 {
+    const DEFAULT_MAX_EXPORT_SIZE_BYTES = 100000000000;
```

Comment:
```suggestion
    const DEFAULT_MAX_EXPORT_SIZE_BYTES = 100000000000; // 100GB
```
There's 100GB there, but I think 10GB was suggested default max? https://keboola.atlassian.net/browse/PS-213

Reply:
Sorry, I forgot to add that comment! So, the problem is how to deploy this because it can potentially break existing jobs. I made this from shared config:

This shows number of tables in % whose size is above the given size
size,above size
1000,0.910371
10000,0.650763
100000,0.500976
1000000,0.376437
10000000,0.256332
100000000,0.141607
1000000000,0.068995
10000000000,0.018081
100000000000,0.002915
1000000000000,0.000311

I.e there are still 1.8% of tables larger than 10GB - so that can have noticeable impact, but there are only 0.2% of tables larger than 100GB which should have negligible impact. Also a 100GB table is probably still exportable (probably a 300-700GB CSV).

Reply:
I'm ok with that (makes sense to me) I just wasn't sure because it's different from the issue description.  I guess it's a question for @padak ?

Reply:
I'll go ahead and merge this, we can lower the default limit anytime later if needed